wicket/src/ui/panes/update.rs

```diff
@@ -460,6 +469,64 @@ impl UpdatePane {
         frame.render_widget(popup, full_screen);
     }
 
+    fn draw_clear_update_state_waiting_popup(
+        &self,
+        state: &State,
+        frame: &mut Frame<'_>,
+    ) {
+        // What we show here depends on the current status.
```

Comment:
Is this comment still accurate? It looks like this only ever shows the "waiting" status, and a different method is responsible for showing errors.

Reply:
Ah, this was removed in #3194

Reply:
I removed them from here as well.